examples/daal/cpp/source/utils/service.h

```diff
@@ -672,36 +672,56 @@ bool checkFileIsAvailable(std::string filename, bool needExit = false) {
     }
 }
 
+inline bool check_file(const std::string &name) {
+    return std::ifstream{ name }.good();
+}
+
+inline std::string get_data_path(const std::string &name) {
```

Comment:
Can you please write a description or some comment?
As far as I understand this was added because with the data folder has different relative path in make and bazel.
But this is not obvious from the code.

Reply:
yes, in general its exactly the copy of the function from oneapi examples, but I will add some comments